src/getTime.js

```diff
@@ -18,7 +17,9 @@
  * @returns {string} - valid time
  */
 function getTime(str) {
-  // write code here
+  const pattern = str.match(/\b([0-1][0-9]|2[0-3]):[0-5][0-9]\b/);
```

Comment:
Слово pattern с английского переводится - шаблон.
Мы можем создать шаблон, по которому будем вести поиск в строке.
`const pattern = /\b([0-1][0-9]|2[0-3]):[0-5][0-9]\b/;`

Когда мы делаем поиск по строке, мы получаем совпадения (их может и не быть)
Совпадения на английском языке будет - matches
`const matches = string.match(pattern);`

Итого:
```suggestion
  const pattern = /\b([0-1][0-9]|2[0-3]):[0-5][0-9]\b/;
  const matches = str.match(pattern);
  
  return matches ? matches[0] : '';
```